source/slang/slang-emit-wgsl.cpp

```diff
@@ -87,9 +87,23 @@ void WGSLSourceEmitter::emitParameterGroupImpl(
     auto varLayout = getVarLayout(varDecl);
     SLANG_RELEASE_ASSERT(varLayout);
 
-    for (auto attr : varLayout->getOffsetAttrs())
+    EmitVarChain blockChain(varLayout);
```

Comment:
we're finally web3

Reply:
> we're finally web3

Speaking of block chains...

Reply:
I am not accountable for this. The code and naming is copied from the glsl implementation. I am curious who did this.